neural_speed/application/main_pybind.cpp

```diff
@@ -82,7 +82,7 @@ void init_gpt_params(gpt_params* params, const std::string& model_path, int max_
                      float temperature = 0.8, int min_new_tokens = 0, float length_penalty = 1.0f,
                      bool early_stopping = false, int n_keep = 0, int n_discard = -1, bool shift_roped_k = false,
                      int batch_size = 1, model_vocab::id pad_token = -1, const std::string& memory_dtype = "auto",
-                     const bool& continuous_batching = false, const int& max_request_num = MODEL_MAX_REQUEST_NUM,
```

Comment:
Isn't it necessary to write this way `const bool&`？https://stackoverflow.com/questions/11932614/in-c-is-it-bad-to-pass-a-const-bool-by-reference

Reply:
Thanks. fixed in https://github.com/intel/neural-speed/pull/145/commits/b709ee965594eb8d70575d14069e69860b720241